eagle-assembly/src/main/conf/eagle.conf

```diff
@@ -75,7 +77,8 @@ metadata {
     username = "root"
     password = ""
     driverClassName = com.mysql.jdbc.Driver
-    url = "jdbc:mysql://server.eagle.apache.org:3306/eagle"
+    database = "eagle"
+    connection = "jdbc:mysql://server.eagle.apache.org:3306/eagle?user=root"
```

Comment:
ya, i noticed that too, but kept whatever was there, let me remove it from connection